### PR TITLE
NEW : Add custom Text on footer total

### PR DESCRIPTION
**Pull request by defrance**
This function allows you to add a small text to each invoice below the total. Just add a miscellaneous parameter to activate it
(to wish the happy new year or an indication of invoicing)

# Instructions
*This is a template to help you make good pull requests. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *only keep the "FIX", "CLOSE" or "NEW" section* (use uppercase to have the PR appears into the ChangeLog, lowercase will not appears)
- *follow the project [contributing guidelines](/.github/CONTRIBUTING.md)*
- *replace the bracket enclosed texts with meaningful information*

# FIX|Fix #[*issue_number Short description*]
[*Long description*]

# CLOSE|Close #[*issue_number Short description*]
[*Long description*]

# NEW|New [*Short description*]
[*Long description*]

### CODE REVIEW

File: htdocs/core/modules/facture/doc/pdf_crabe.modules.php [hunk 1]
```diff
@@ -1646,6 +1647,14 @@ protected function _tableau_tot(&$pdf, $object, $deja_regle, $posy, $outputlangs
 		}
 
 		$index++;
+
+		if (!empty($conf->global->BILL_TEXT_TOTAL_FOOTER)) {
```
Comment (eldy) [inline htdocs/core/modules/facture/doc/pdf_crabe.modules.php:1651]: You should use now getDolGlobalString('...') instead of conf->global->...

**Review by defrance — COMMENTED** [review]
OKI change made

File: htdocs/core/modules/facture/doc/pdf_crabe.modules.php [hunk 2]
```diff
@@ -1646,6 +1647,14 @@ protected function _tableau_tot(&$pdf, $object, $deja_regle, $posy, $outputlangs
 		}
 
 		$index++;
+
+		if (getDolGlobalString($conf->global->BILL_TEXT_TOTAL_FOOTER)) {
```
Comment (eldy) [inline htdocs/core/modules/facture/doc/pdf_crabe.modules.php:1651]: Should be
`if (getDolGlobalString('BILL_TEXT_TOTAL_FOOTER')) {`

Comment (defrance) [reply htdocs/core/modules/facture/doc/pdf_crabe.modules.php]: sorry

**Pull request merged**